app/components/forms/order-form.js

```diff
@@ -0,0 +1,170 @@
+        }
+      ]
+    };
+    let emailValidation = {
+      rules: [
+        {
+          type   : 'empty',
+          prompt : this.i18n.t('Please enter your email')
+        }
+      ]
+    };
+    let validationRules = {
+      inline : true,
+      delay  : false,
+      on     : 'blur',
+      fields : {
+        firstname: {
+          identifier : 'first_name',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.i18n.t('Please enter your first name')
+            }
+          ]
+        },
+        lastname: {
+          identifier : 'last_name',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.i18n.t('Please enter your last name')
+            }
+          ]
+        },
+        email: {
+          identifier : 'email',
+          rules      : [
+            {
+              type   : 'email',
+              prompt : this.i18n.t('Please enter a valid email address')
+            }
+          ]
+        },
+        country: {
+          identifier : 'country',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.i18n.t('Please enter your country')
+            }
+          ]
+        },
+        address: {
+          identifier : 'address',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.i18n.t('Please enter your address')
+            }
+          ]
+        },
+        city: {
+          identifier : 'city',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.i18n.t('Please enter your city ')
+            }
+          ]
+        },
+        state: {
+          identifier : 'state',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.i18n.t('Please enter your state')
+            }
+          ]
+        },
+        zipcode: {
+          identifier : 'zip_code',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.i18n.t('Please enter your zip code')
+            }
+          ]
+        },
+        payvia: {
+          identifier : 'pay_via',
+          rules      : [
+            {
+              type   : 'checked',
+              prompt : this.i18n.t('Please specify your choice of payment method ')
+            }
+          ]
+        }
+      }
+    };
+    this.holders.forEach((value, index) => {
+      validationRules.fields[`first_name_${index}`] = firstNameValidation;
+      validationRules.fields[`last_name_${index}`] = lastNameValidation;
+      validationRules.fields[`email_${index}`] = emailValidation;
+    });
+    return validationRules;
+  },
+  countries: computed(function() {
+    return countries;
+  }),
+  actions: {
+    submit() {
+      this.onValid(() => {
+      });
+    },
+    fillHolderData(holder, sameAsBuyer ) {
+      if(isBuyer === sameAsBuyer) {
```

Comment:
@niranjan94 based on your suggestions I tried to implement but not getting the desired output. Please review and let me know where I am wrong. Thanks.